contracts/exchange/api/TransferContract.sol

```diff
@@ -11,6 +11,22 @@ import {IERC20MetadataUpgradeable} from "@openzeppelin/contracts-upgradeable/tok
 abstract contract TransferContract is TradeContract {
   using BIMath for BI;
 
+  event Withdrawal(
+    address indexed fromAccount,
+    address indexed recipient, // the recipient of the withdrawal on L1
+    Currency currency,
+    uint64 numTokens,
+    uint64 txID
+  );
+
+  event Deposit(
```

Comment:
@minhbsq is this event for deposit confirmation?

Reply:
Yes it is, to be doubly sure that we don't miss any events in the historical deposit API I include it here. 
For day 1 I might not use it yet (unless there's enough time to add it to my implementation)